app/views/layouts/_topbar.html.haml

```diff
@@ -3,7 +3,7 @@
     %li.name
 
       %h1#shostakodech
-        %a#home{:href => root_path} Shostakodech
+        %a.home{:href => root_path} Shostakodech
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace